controllers/site.js

```diff
@@ -0,0 +1,125 @@
+const Site = require('../models/Site');
+
+/**
+ * POST /sites
+ * Create a new site
+ */
+exports.postSite = (req, res, next) => {
+  req.assert('title', 'Title cannot be blank').notEmpty();
+  req.assert('url', 'Url cannot be blank').notEmpty();
+
+  const errors = req.validationErrors();
+
+  if (errors) {
+    res.json(errors);
+  } else {
+    const owner = req.user._id;
+    const members = [owner];
+    const { url, title } = req.body;
+
+    new Site({
+      title,
+      url,
+      owner,
+      members
```

Comment:
to make git diff cleaner, can we use additional trailing comma ?
https://github.com/airbnb/javascript#commas--dangling